WorkbenchProject/src/main/java/edu/uwb/braingrid/workbench/ui/LoginCredentialsDialog.java

```diff
@@ -37,6 +41,31 @@ private void initComponents() {
         hostnameLabel.setText("Hostname: ");
 
         usernameLabel.setText("Username: ");
+        File inputFile = new File(System.getProperty("user.dir") + "\\Cache\\username.encrypted");
+        File key = new File(System.getProperty("user.dir") + "\\Key");
+		try {
+			FileInputStream keyInput = new FileInputStream(key);
+	        ObjectInputStream keyObj;
+			try {
+				keyObj = new ObjectInputStream(keyInput);
+				String keyString;
+				try {
+					keyString = (String) keyObj.readObject();
+				    SimulationSpecificationDialog tempDialog
+				    		=
+				    		new SimulationSpecificationDialog();
+				    String textName
+				    	=
+				    	tempDialog.decrypt(keyString, inputFile,
+				    			inputFile, "username");
+				    usernameTextField.setText(textName);
+				}catch (ClassNotFoundException e) {
+					e.printStackTrace();
+			} 
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+	} catch (FileNotFoundException e) {}
```

Comment:
For this exception, since when a new user use it for the first time, no cache is available and this exception will be thrown. But I don't wan't to cerr anything since it's expected